client/checkout/style.scss

```diff
@@ -3,27 +3,46 @@
 }
 
 .js-woopayments-copy-test-number {
-	line-height: 1.2em;
```

Comment:
Since the markup changed, the styles needed to change as well.
Before, the `button` element didn't have any text, so we didn't really care what it looked like.
Now, we need to ensure the text within the `button` element plays nice across different themes and browsers.

So I reset the `padding`, changed the `font-size` and `font-weight`, ensured the `cursor` was set to `pointer` (I dunno why, but it wasn't before by default), and text color and background color matched the surrounding text.